redbeams-api/src/main/java/com/sequenceiq/redbeams/api/model/common/Status.java

```diff
@@ -88,8 +88,13 @@ public boolean isStartInProgressOrCompleted() {
     }
 
     public boolean isUpgradeInProgress() {
-        return UPGRADE_REQUESTED.equals(this) ||
-                UPGRADE_IN_PROGRESS.equals(this);
+        return UPGRADE_REQUESTED.equals(this)
+                || UPGRADE_IN_PROGRESS.equals(this);
+    }
+
+    public boolean isUpgradeAvailable() {
```

Comment:
The name "is upgrade available" I think means rather when there is an upgrade for your database server to apply. I would thus rename this status as isUpgradeAllowed or similar.